### ISSUE TITLE
Color contrast between `brand-ripe` and `off-white` doesn't meet AA standard

### ISSUE THREAD

**Issue by mikenikles** [gitpod-io/website#118]
Check at https://color.a11y.com.

Color values can be found in `src/assets/scss/_variables.scss`

**Comment by nisarhassan12** [comment]
The Gitlab and Bitbucket buttons are the ones who don't meet the standard:

![image](https://user-images.githubusercontent.com/46004116/124862470-ded9ba00-dfce-11eb-9a4d-c7c865053808.png)

**Comment by ChristinFrohne** [comment]
What about these colors? They are part of the color theme provided by MAD
#5E48BA
#58B19F (alternative #265583)
<img width="296" alt="Bildschirmfoto 2021-07-09 um 15 31 44" src="https://user-images.githubusercontent.com/46345125/125085418-d9619a00-e0ca-11eb-83f9-b745bb96b3b6.png">

**Comment by nisarhassan12** [comment]
`#58B19F` didn't pass the accessibility audit so I used the alternate that you proposed.
